packages/page-accounts/src/modals/Create.tsx

```diff
@@ -64,12 +66,12 @@ const STEPS_COUNT = 3;
 function getSuri (seed: string, derivePath: string, pairType: PairType): string {
   return pairType === 'ed25519-ledger'
     ? u8aToHex(hdLedger(seed, derivePath).secretKey.slice(0, 32))
-    : `${seed}${derivePath}`;
+    : pairType === 'ethereum' ? `${seed}/${derivePath}` : `${seed}${derivePath}`;
```

Comment:
```suggestion
    : pairType === 'ethereum'
      ? `${seed}/${derivePath}`
      : `${seed}${derivePath}`;
```

Just since we are using indents above, probably looks clearer. (At least to my eyes)